components/nimbus/android/build.gradle

```diff
@@ -74,10 +68,21 @@ configurations {
     jnaForTest
 }
 
+ext.gleanYamlFiles = ["${project.projectDir}/../metrics.yaml"]
+ext.gleanGenerateMarkdownDocs = true
+ext.gleanDocsDirectory = "$rootDir/docs/metrics/nimbus"
+apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
+
 dependencies {
     jnaForTest "net.java.dev.jna:jna:$jna_version@jar"
     implementation "net.java.dev.jna:jna:$jna_version@aar"
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    implementation "androidx.annotation:annotation:$androidx_annotation_version"
+
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlin_coroutines_version"
+
+    implementation "org.mozilla.components:service-glean:$android_components_version"
```

Comment:
I wasn't aware that there are other modules here that also depend on `services-glean`. I thought `concept-fetch` was the only dependency (#3872). Potentially also something to rethink, but out of scope for this PR.

Reply:
FWIW, we're very motivated to take this glean dependency at the Rust level rather than the Kotlin level, and I'm hopeful that when that work comes to fruition it will break the dependency cycle here.